nengo/conftest.py

```diff
@@ -56,7 +56,7 @@ def load_class(fully_qualified_name):
 
 @pytest.fixture(scope="session")
 def Simulator(request):
-    """the Simulator class being tested.
+    """The Simulator class being tested.
```

Comment:
FYI this used to be the convention used by py.test for fixture docstrings (which are printed when you do `py.test --fixtures`) but it seems as though they've switched to sentence case :+1: